ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -449,11 +451,13 @@ export default class ConfirmTransactionBase extends Component {
                     />,
                   ])}
                   subTitle={
-                    <GasTiming
```

Comment:
This actually doesn't need to happen in this PR and will be removed when I correctly rebase. But It is okay to have it here for now, so that the error it throws in not a distraction for QA. It has been fixed properly on another PR that will soon be merged

- [ ] Remove change after rebasing onto develop